docs/open-source-project/release-notes/tizen-5-0-m1.md

```diff
@@ -0,0 +1,952 @@
+
+​		**o**   **Supported for lock/unlock functions has been added**
+
+​	**·**        **Multipath TCP (In-house only module)**
+
+​		**o**   **New feature for multipath TCP support has been added**
+
+​			**§**  **A connection API (In-house only) for enabling the multipath TCP function has been added**
+
+​		**§**  **A connection API (In-house only) for setting the path-manager for multipath TCP has been added**
+
+​			**§**  **A connection API (In-house only) for setting the scheduling for multipath TCP has been added**
+
+​	**·**        **TCP Fast Open**
+
+​		**o**   **A http API for allowing TCP Fast Open has been added**
+
+​	**·**        **SoftAP**
+
+​		**o**   **capi-network-softap has been open to the public**
+
+​	**·**        **MTP FunctionFS**
+
+​		**o**   **FunctionFS has been added in USB File System which is based on MTP operation**
+
+​	**·**        **Wi-Fi Native CAPI**
+
+​		**o**   **capi-wifi has been removed and instead, capi-wifi-manager has been replaced**
+
+​		**o**   **New features and APIs** in mobile, wearable, and TV profiles**:**
+
+​			§  **A wifi-manager** API for getting the all VSIE(**Vendor Specific Information Elements)** of AP has been added.
+
+​			§  **A wifi-manager** API for getting the WPS PIN number has been added.
+
+​			§  **A wifi-manager** API for getting the country code has been added.
+
+​			§  **A wifi-manager** API for getting the BSSID list has been added.
+
+​			§  **Supported for** EAP-FAST, EAP-PWD and EAP-AKA-PRIME has been added.
+
+​	**·**        **Wi-Fi Direct Native CAPI**
+
+​		**o**   **New features and APIs** in mobile, wearable, and TV profiles**:**
+
+​			**§**  **A wifi-direct API for getting the peer VSIE(Vendor Specific Information Elements) has been added**
+
+​			**§**  **A wifi-direct API for creating Wi-Fi direct group with SSID has been added**
+
+​			**§**  **A wifi-direct API for setting/getting the intent of the group owner for each connection type has been added**
+
+​			**§**  **A wifi-direct API for set/unset the notification for peer device info connection state has been added**
+
+​			**§**  **A wifi-direct API for setting/getting the WPS config method has been added**
+
+​			**§**  **A wifi-direct API for removing the persistent device or devices have been added**
+
+​	**·**        **Telephony**
+
+​		**o**   **Voicecall state based API has been removed.**
+
+​		**o**   **Videocall state based API has been removed.**
+
+​		**o**   **Poweroff wait sequence for CP detach has been applied.**
+
+​	**·**        **Bluetooth CAPI**
+
+​		**o**   **Old GATT APIs have been removed, these APIs were deprecated in Tizen 2.3.1**
+
+​		**o**   **New features and APIs** in mobile, wearable, and TV profiles**:**
+
+​			§  **A delay report API for AVRCP streaming (in-house only)** has been added.
+
+​		**o**   **A callback API for AVRCP delay changed (in-house only)** has been added.
+
+​	**·**        **Open Source Upgrade**
+
+​		**o**   **wpa_supplicant: 2.5** **à** **2.6.**

```

Comment:
is it 2.5 to 2.6?

Reply:
It is from 2.4 (@tizen 4.0) to 2.6 (@tizen 5.0).